docs/calendar-review.md

```diff
@@ -1,36 +1,36 @@
 # Supporting Calendar Systems in Temporal
 
-[ECMAScript Temporal](https://github.com/tc39/proposal-temporal/) is the new date and time API for ECMAScript.  Because ECMAScript is designed for global audiences, high-quality internationalization is a design goal.
+[ECMAScript Temporal](https://github.com/tc39/proposal-temporal/) is the new date and time API for ECMAScript. Because ECMAScript is designed for global audiences, high-quality internationalization is a design goal.
 
 This document discusses why and how calendar systems are included in the Temporal API, explains problems we expect developers to encounter with calendars, and outlines how to avoid those problems. A final section explains alternative designs for using calendar systems in Temporal, along with an explanation of why the current approach was chosen instead.
 
 ## Background and Motivation
 
 Calendar systems provide a way to assign human-readable numbers, such as year, month, and day, to a particular period of time.
 
-Much of the world uses the Gregorian calendar system, which uses solar years, months, and days: over time, it remains synchronized with the cycle of the seasons.  However, there are other calendar systems used throughout much of the world.  Some popular calendar systems include Hebrew, Islamic, Buddhist, Hindu, Chinese, Japanese, and Ethiopic.
+Much of the world uses the Gregorian calendar system, which uses solar years, months, and days: over time, it remains synchronized with the cycle of the seasons. However, there are other calendar systems used throughout much of the world. Some popular calendar systems include Hebrew, Islamic, Buddhist, Hindu, Chinese, Japanese, and Ethiopic.
 
-Today's date is March 4, 2021 in the Gregorian calendar.  Here are several other representations of that date:
+Today's date is March 4, 2021 in the Gregorian calendar. Here are several other representations of that date:
 
-*   20 Adar 5781 (Hebrew)
-*   Rajab 20, 1442 AH (Islamic, Umm al-Qura)
-*   March 4, 2564 BE (Thai Buddhist)
-*   令和3年3月4日 == March 4, 3 Reiwa (Japanese)
+- 20 Adar 5781 (Hebrew)
+- Rajab 20, 1442 AH (Islamic, Umm al-Qura)
+- March 4, 2564 BE (Thai Buddhist)
+- 令和 3 年 3 月 4 日 == March 4, 3 Reiwa (Japanese)
```

Comment:
The output before was the exact string from ICU. The spaces help for readability, but they're not technically correct

Reply:
This looks like https://github.com/prettier/prettier/issues/6385. I'll see if there's a good way to work around this bug without completely turning off prettier for this file.

Reply:
Wrapped it with `<!-- prettier-ignore-start -->` and `<!-- prettier-ignore-end -->` while leaving the rest intact.